packages/components/src/components/icon/common/ic-cat-turbos.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M0 14h1.667v1H0v-1Zm3.667 0h1.666v1H3.667v-1Zm3.666 0H9v1H7.333v-1ZM12 13h4v3h-4l-1.5-1.5L12 13Z" fill="var(--fill-color1)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M0 .5A.5.5 0 0 1 .5 0H9v1H.5A.5.5 0 0 1 0 .5Z" fill="var(--fill-color1)"/><path d="M15.5 0a.5.5 0 0 1 .5.5v5a.5.5 0 0 1-1 0V1.707L6.854 9.854a.5.5 0 0 1-.708 0L2.5 6.207 0 8.707V7.293l2.146-2.147a.5.5 0 0 1 .708 0L6.5 8.793 14.293 1H11V0h4.5Z" fill="var(--fill-color1)"/></svg>
```

Comment:
--fill values are already a equivalent to a var so no need to recalculate them again.
ref: packages/components/src/components/icon/icon.scss

```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M0 14h1.667v1H0v-1Zm3.667 0h1.666v1H3.667v-1Zm3.666 0H9v1H7.333v-1ZM12 13h4v3h-4l-1.5-1.5L12 13Z" fill="var(--fill-color1)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M0 .5A.5.5 0 0 1 .5 0H9v1H.5A.5.5 0 0 1 0 .5Z" fill="--fill-color1"/><path d="M15.5 0a.5.5 0 0 1 .5.5v5a.5.5 0 0 1-1 0V1.707L6.854 9.854a.5.5 0 0 1-.708 0L2.5 6.207 0 8.707V7.293l2.146-2.147a.5.5 0 0 1 .708 0L6.5 8.793 14.293 1H11V0h4.5Z" fill="--fill-color1"/></svg>
```